app/controllers/concerns/update_multiple_transcripts_concern.rb

```diff
@@ -0,0 +1,18 @@
+module UpdateMultipleTranscriptsConcern
+  extend ActiveSupport::Concern
+
+  def publish_trancripts(transcript_ids)
+    @collection.transcripts.where(uid: transcript_ids).each { |t| t.publish! }
```

Comment:
better to use `find_each`? although i'm not sure how big the ids is.

and it can be shorten to:

```ruby
@collection.transcripts.where(uid: transcript_ids).find_each(&:publish!)
```